### ISSUE TITLE
`minADE` During Training

### ISSUE THREAD

**Issue by MichaelYu781** [stepankonev/waymo-motion-prediction-challenge-2022-multipath-plus-plus#11]
hi, @stepankonev!
I have trained the model for about 40 epochs, using this [configuration](https://github.com/stepankonev/waymo-motion-prediction-challenge-2022-multipath-plus-plus/blob/cf020ffa5ea451389fc43d4fc460cb22ba2cc124/code/configs/final_RoP_Cov_Single.yaml). An evaluation of `minADE` on the validation set of `waymo motion`  is made , and the picture below is the outcome. Is the change of this metrics normal? Other configurations like setting `trainable_cov = false` have also been tried, and get analogous outcome on the change of `minADE`.
Thanks.

![test1](https://user-images.githubusercontent.com/78360527/181428727-d2aa173d-6299-4c39-8a89-b714a434d00d.png)

**Comment by stepankonev** [comment]
Hello! Could you clarify what you mean by `change` please? Basically speaking I mot sure that I have the track of minADE while training.

**Comment by Jerrrrykun** [comment]
@MichaelYu781 Could you explain what does the y-axis represent in your plot? I just split the `training` folder of the `waymo motion dataset` into (90%-10%) train-dev sets. But when I evaluate the single decoder model on the dev set with about 100, 000 training steps, the `minADE` is still above 16+ meters.

**Comment by MichaelYu781** [comment]
Hi, @Jerrrrykun. I follows the definition `https://waymo.com/open/challenges/2022/motion-prediction/` here.

**Comment by Jerrrrykun** [comment]
> Hi, @Jerrrrykun. I follows the definition `https://waymo.com/open/challenges/2022/motion-prediction/` here.

Thanks for your reply. I am using the same definition in my own implementation, too. I have solved it. My problem lies in the normalization part.

**Comment by Terra-V** [comment]
> 

hi, @Jerrrrykun.  May you further explain the solution of the large minADE problem? I have trained the model with more than 120 epochs, but the minADE result is even worse than what you previously got.

**Comment by lsrrrrrrrrr** [comment]
> Hi, [@Jerrrrykun](https://github.com/Jerrrrykun). I follows the definition `https://waymo.com/open/challenges/2022/motion-prediction/` here.
Same problem, how do you solve it?
